client/web/src/repo/RepoContainer.tsx

```diff
@@ -180,6 +187,30 @@ export const RepoContainer: FC<RepoContainerProps> = props => {
         )
     )
 
+    /**
+     * Cody client.
+     */
+    const isCodyEnabled = useFeatureFlag('cody-experimental')
```

Comment:
```suggestion
    const [isCodyEnabled] = useFeatureFlag('cody-experimental')
```

useFeatureFlag returns a tuple 😱

Reply:
Fixed also in the code that you based this on: https://github.com/sourcegraph/sourcegraph/pull/50635/files

Reply:
😆 - Thanks, Phillipp.